app/models/message.rb

```diff
@@ -1,6 +1,13 @@
 class Message < ApplicationRecord
-
   belongs_to :member
   belongs_to :room
 
+  validates :body, presence: true
+
+  def self.message_create!(current_member, message_params)
+    message = Message.new(message_params)
```

Comment:
クラスメソッド内で自身に処理を行いたいなら下記のように実装できます

```suggestion
    message = new(message_params)
```

Reply:
ただ、 https://github.com/shushikata/medi_navi/pull/65/files#r547563950 のコメントの通り、このメソッドは不要かと思います